js/plugins/controlPanel.js

```diff
@@ -12,7 +13,7 @@ Genoverse.Plugins.controlPanel = function () {
         icon    : '<i class="fas fa-chevron-right"></i>',
         'class' : 'gv-scroll-right'
       }],
-      init: function (browser) {
+      init: function (browser) {        
```

Comment:
Trailing spaces here
```suggestion
      init: function (browser) {
```

Reply:
Changed it